src/firebase/async-firestore.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase/app'
+
+let loaded = false
+
+export default async () => {
+  if (!loaded) {
+    await import('firebase/firestore')
```

Comment:
Use prefetch to load as soon as possible
```suggestion
    await import(/* webpackPrefetch: true */ 'firebase/firestore')
```

Reply:
In my opinion, we don't need to do that. In the generated html we have already that :

```
<link href="/chunk-ea7a42bc.js" rel="prefetch">
```

It indicates to the browser to download this resource as soon as it can with the lowest priority.

Reply:
does `chunk-ea7a42bc.js` correspond to the firestore package ?